### PR TITLE
docs: add decorators across all notebooks

### PR DESCRIPTION
**Pull request by hassiebp**
EDIT: takeover of this PR for notebooks edited by @ChrisTho23 

@hassiebp your commits moved to #444 (main working branch for decorator relaunch of docs)

### CODE REVIEW

File: cookbook/integration_langchain.ipynb [hunk 1]
```diff
@@ -378,15 +378,19 @@
     },
```
Comment (marcklingen) [inline cookbook/integration_langchain.ipynb:378]: <p>lets focus on this example.</p><p> </p><p>add single line comments to update_current_trace and get_current_langchain_handler </p><p> </p><p>remove params from main function, just execute it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='34'/>

Comment (marcklingen) [inline cookbook/integration_langchain.ipynb:378]: <p>remove this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='35'/>

Comment (marcklingen) [inline cookbook/integration_langchain.ipynb:378]: <p>let's remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='36'/>

Comment (marcklingen) [inline cookbook/integration_langchain.ipynb:378]: <p>function name = <code>name</code></p><p> </p><p>remove  <code>update_current_trace</code></p><p> </p><p>add single line comments to get_current_langchain_handler and   chain.invoke</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='43'/>

Comment (marcklingen) [inline cookbook/integration_langchain.ipynb:378]: <p>let's remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='44'/>

File: cookbook/integration_openai_sdk.ipynb [hunk 2]
```diff
@@ -14,14 +14,18 @@
     },
```
Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <p>remove output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <p>before committing, remove outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='18'/>

Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <p>remove output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <p>we do not talk about <code>users</code> </p><p> </p><p>"Optionally, you can pass your own trace_id to langfuse if you want to sync it with your internal ids. Just pass it as a <code>langfuse_observation_id</code> kwarg. </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='24'/>

Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <pre class="ql-syntax" spellcheck="false">Line #1.    from langfuse.decorators import langfuse_context, observe
</pre><p>remove observe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='25'/>

Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <p>switch to decorators here as well for trace creation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/#comment-1526660692'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='32'/>

**Comment by ChrisTho23** [comment]
Please replace screenshots @marcklingen. Links to traces are pasted into the notebooks

File: cookbook/integration_langchain.ipynb (hunk 1 above)
Comment (marcklingen) [inline cookbook/integration_langchain.ipynb:378]: <p>duplicate of comment below and does not match <code>get_current_langchain_handler</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/#comment-1527191713'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='30'/>

File: cookbook/integration_openai_sdk.ipynb (hunk 2 above)
Comment (marcklingen) [inline cookbook/integration_openai_sdk.ipynb:14]: <pre class="ql-syntax" spellcheck="false">Line #1.    from langfuse.decorators import langfuse_context, observe
</pre><p>i'd include the import statement again just for clarity</p><p>from langfuse.openai import openai</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/#comment-1527191714'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

**Review by marcklingen — APPROVED** [review]

**Pull request merged**